app/views/home/summit.html.erb

```diff
@@ -108,6 +108,7 @@
     </div>
       <div class="sponsor-logos">
         <%= link_to image_tag('summit/turing-horizontal.png', height: '100', alt: 'turing.io'), 'https://www.turing.io/', target: '_blank' %>
+        <%= link_to image_tag('sponsors/IBM.png', height: '100', alt: ‘IBM’), 'https://www.ibm.com', target: '_blank' %>
```

Comment:
are these special characters around 'IBM'? the quotes seem different...copy/pasted in?